src/ddoc/macros.d

```diff
@@ -502,8 +502,11 @@ private bool parseAsKeyValuePair(ref Lexer olexer, ref string key, ref string va
 	}
 	else
 		return false;
-	while (!lexer.empty && lexer.front.type == Type.whitespace)
+
+	// parentheses might be used in the key
+	while (!lexer.empty && lexer.front.type.among(Type.whitespace, Type.lParen, Type.rParen, Type.word))
```

Comment:
One also might argue that the occurrence at Phobos should be fixed:

https://github.com/dlang/phobos/blob/548630d2a9ba95f079193d7b47ed8cd685c0f7c8/std/regex/package.d#L353